### ISSUE TITLE
Repositories not being created

### ISSUE THREAD

**Issue by davidcodesido** [appscode/stash#589]
After installing stash via helm I see the following error in k8s apiserver.

```
E0928 06:47:52.839438       1 available_controller.go:311] v1alpha1.repositories.stash.appscode.com failed with: Operation cannot be fulfilled on apiservices.apiregistration.k8s.io "v1alpha1.repositories.stash.appscode.com": the object has been modified; please apply your changes to the latest version and try again
E0928 06:47:53.037530       1 upgradeaware.go:310] Error proxying data from client to backend: read tcp X.X.X.X:6443->X.X.X.X:51410: read: connection reset by peer
I0928 06:47:56.041142       1 controller.go:105] OpenAPI AggregationController: Processing item v1alpha1.admission.stash.appscode.com
E0928 06:47:57.259395       1 controller.go:111] loading OpenAPI spec for "v1alpha1.admission.stash.appscode.com" failed with: failed to retrieve openAPI spec, http error: ResponseCode: 503, Body: service unavailable
, Header: map[Content-Type:[text/plain; charset=utf-8] X-Content-Type-Options:[nosniff]]
I0928 06:47:57.259428       1 controller.go:119] OpenAPI AggregationController: action for item v1alpha1.admission.stash.appscode.com: Rate Limited Requeue.
I0928 06:47:58.259504       1 controller.go:105] OpenAPI AggregationController: Processing item v1alpha1.repositories.stash.appscode.com
E0928 06:47:59.326262       1 controller.go:111] loading OpenAPI spec for "v1alpha1.repositories.stash.appscode.com" failed with: failed to retrieve openAPI spec, http error: ResponseCode: 503, Body: service unavailable
, Header: map[Content-Type:[text/plain; charset=utf-8] X-Content-Type-Options:[nosniff]]
I0928 06:47:59.326286       1 controller.go:119] OpenAPI AggregationController: action for item v1alpha1.repositories.stash.appscode.com: Rate Limited Requeue.
I0928 06:48:00.326280       1 controller.go:105] OpenAPI AggregationController: Processing item v1alpha1.admission.stash.appscode.com
E0928 06:48:00.326532       1 controller.go:111] loading OpenAPI spec for "v1alpha1.admission.stash.appscode.com" failed with: failed to retrieve openAPI spec, http error: ResponseCode: 503, Body: service unavailable
, Header: map[Content-Type:[text/plain; charset=utf-8] X-Content-Type-Options:[nosniff]]
I0928 06:48:00.326544       1 controller.go:119] OpenAPI AggregationController: action for item v1alpha1.admission.stash.appscode.com: Rate Limited Requeue.
I0928 06:48:04.743014       1 controller.go:105] OpenAPI AggregationController: Processing item v1alpha1.admission.stash.appscode.com
I0928 06:48:08.363000       1 controller.go:116] OpenAPI AggregationController: action for item v1alpha1.admission.stash.appscode.com: Requeue.
I0928 06:48:08.363062       1 controller.go:105] OpenAPI AggregationController: Processing item v1alpha1.repositories.stash.appscode.com
I0928 06:48:10.555575       1 controller.go:116] OpenAPI AggregationController: action for item v1alpha1.repositories.stash.appscode.com: Requeue.
```

Stash installed via:

helm upgrade stash-operator appscode/stash --install --namespace stash --set apiserver.ca="$(onessl get kube-ca)" --set apiserver.enableValidatingWebhook=true --set apiserver.enableMutatingWebhook=true --set rbac.create=true

 helm version
Client: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}

onessl version
Version = 0.7.0
VersionStrategy = tag
Os = linux
Arch = amd64
CommitHash = c00f03ceac380a7634eea5a3a889ce0bb8af76ce
GitBranch = release-0.7
GitTag = 0.7.0
CommitTimestamp = 2018-07-18T16:29:17

kubectl version
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.2", GitCommit:"bb9ffb1654d4a729bb4cec18ff088eacc153c239", GitTreeState:"clean", BuildDate:"2018-08-07T23:17:28Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.2", GitCommit:"bb9ffb1654d4a729bb4cec18ff088eacc153c239", GitTreeState:"clean", BuildDate:"2018-08-07T23:08:19Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}

**Comment by davidcodesido** [comment]
Thanks for your prompt answer. Do you have an estimated release date? Is there any beta channel for helm so I can test the version before the final release?

**Comment by hossainemruz** [comment]
We don't have any eta for release. Our engineering team currently working on a different project. We will return to stash as soon as possible. We are looking for second or third week of October.

**Comment by davidcodesido** [comment]
Kubernetes 1.12 is already out. Just in case you want to have a look at the VolumeSnapshots they include: https://kubernetes.io/docs/concepts/storage/volume-snapshots

**Comment by hossainemruz** [comment]
@davidcodesido This problem should be solved in `Stash 0.8.3`. If not, feel free to reopen.
